demisto_sdk/commands/content_graph/objects/pack.py

```diff
@@ -162,15 +161,23 @@ def depends_on(self) -> List["RelationshipData"]:
             if r.content_item_to.database_id == r.target_id
         ]
 
+    def marketplace_compatible_content_items(
+        self, marketplace: MarketplaceVersions
+    ) -> PackContentItems:
+        content_items = [
+            content_item
+            for content_item in self.content_items
+            if marketplace in content_item.marketplaces
```

Comment:
is `content_item.marketplaces` always populated and correct?

Reply:
yes